generators/server/files.js

```diff
@@ -896,6 +896,15 @@ const serverFiles = {
                 },
             ],
         },
+        {
+            path: SERVER_TEST_SRC_DIR,
+            templates: [
+                {
+                    file: 'package/IntegrationTest.java',
+                    renameTo: generator => `${generator.testDir}/${generator.mainClass}IntegrationTest.java`,
```

Comment:
Not sure about my comment, but I think it's better to have simply IntegrationTest, as some project can have very long name, and this annotation will be so long.
What do you think ?

Reply:
@pascalgrimaud Yes I agree annotation will be quite long for some (for e.g. `@LibraryManagementSystemIntegrationTest` is quite long), and package name provides enough namespace info already (`com.librarymanagement.app.IntegrationTest`) , so we can go with just `@IntegrationTest`

Reply:
ok then, let's change to `@IntegrationTest`

Reply:
@pascalgrimaud  annotation refactored to `@IntegrationTest`. requested review